### DISCUSSION TITLE
Hero 12 set Photo Interval Mode

### DISCUSSION THREAD

**Discussion by saustinp** [gopro/labs#889] [Q&A]
Hi,

I just bought a new Hero 12 and am excited to get set up with GP labs. I'm trying to get the "Interval" Photo mode running with Labs -- here are my settings: https://gopro.github.io/labs/control/set/?cmd=mTPp.5fW0rdR0oT240504161441

The camera reads the QR code (check box appears on the screen) but does not adjust the settings accordingly. I can manually set these settings however, so I'm not sure what's going on here, any ideas?

Thanks!
Sam

**Comment by dnewman-gpsw** [comment]
There is no Labs command to enable photo interval, and you have request Timelapse Photo, which is a different mode (and requires you have an existing Timelapse Photo preset.)

**Comment by saustinp** [comment]
Got it, thank you! Is there a plan to add support for photo interval to Labs?

**Comment by euromarco** [comment]
Hi, I would also like to have controls to set interval and (self)timer
